comp/core/agenttelemetry/impl/sender.go

```diff
@@ -343,8 +343,8 @@ func (ss *senderSession) flush() Payload {
 	if len(ss.metricPayloads) == 1 {
 		// Single payload will be sent directly using the request type of the payload
 		mp := ss.metricPayloads[0]
-		payload.RequestType = "agent-metrics"
-		payload.Payload = payloadInfo{"agent-metrics", mp}.payload
```

Comment:
This and will not work. Semantics of the agent-metrics here implied metric telemetry payloads from Agent for the Instrumentation Telemetry intake. Intake expect (read hardcoded) only this type of request type (we do have another request type agent-bsod).